lib/smart_answer/flow.rb

```diff
@@ -14,6 +14,7 @@ def self.build
 
     def initialize(&block)
       @nodes = []
+      status(:draft)
```

Comment:
If the default is draft? Do we need to add a `:published` flag to all existing smart answers?

Reply:
They already all have this